### PR TITLE
Latest changes sdk modelmonitor

### PR DESCRIPTION
**Pull request by pallavi-pannu-oc**

### CODE REVIEW

File: dkube/sdk/api.py [hunk 1]
```diff
@@ -2882,6 +2882,13 @@ def modelmonitor_update_alert(
         alert_dict["class"] = alert_dict.pop("_class")
         return super().update_modelmonitor_alert(id, alert_id, alert_dict)
 
+    def remove_none(self,obj):
```
Comment (songole) [inline dkube/sdk/api.py:2885]: Is this used anywhere?

File: dkube/sdk/api.py [hunk 2]
```diff
@@ -2924,7 +2927,37 @@ def modelmonitor_update(
         [config_dict.pop(key) for key in rem_list]
         for k in list(config_dict.keys()):
             if config_dict[k] == None or config_dict[k] == []:
-                del config_dict[k]
+                 del config_dict[k] 
+       
+        if 'alerts' in config_dict.keys():
+            all_alerts = super().get_modelmonitor_alerts(id)
+            for i in all_alerts:
```
Comment (songole) [inline dkube/sdk/api.py:2934]: Remove alert update from here. We have update API for alerts. Users should use it.

Comment (pallavi-pannu-oc) [reply dkube/sdk/api.py:2934]: @songole for datasets also, we have a update api, should i remove that too..?

File: dkube/sdk/api.py (hunk 1 above)
Comment (pallavi-pannu-oc) [reply dkube/sdk/api.py:2885]: this i will remove..

**Pull request merged**